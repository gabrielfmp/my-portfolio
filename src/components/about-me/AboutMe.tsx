import { AboutMeWrapper, StyledTitle } from './styles';
import { Contact } from '../contact';
import CakeSvg from '../../ui/iconography/cake.svg';
import EmailSvg from '../../ui/iconography/email.svg';
import GithubSvg from '../../ui/iconography/github.svg';
import LinkedInSvg from '../../ui/iconography/linkedin.svg';
import LocationSvg from '../../ui/iconography/location.svg';
import PhoneSvg from '../../ui/iconography/phone.svg';
import React from 'react';

const AboutMe = () => {
  return (
    <AboutMeWrapper>
      <StyledTitle>About me</StyledTitle>
      <Contact
        icon={EmailSvg}
        iconText="email"
        contact="gabrielfmpires@gmail.com"
      />
      <Contact
        icon={PhoneSvg}
        iconText="phone number"
        contact="+351 934429387"
      />
      <Contact
        icon={LocationSvg}
        iconText="location"
        contact="Lisbon, Portugal"
      />
      <Contact icon={CakeSvg} iconText="birthday" contact="17 May 1997" />
      <Contact icon={GithubSvg} iconText="github" contact="gabrielfmp" />
      <Contact
        icon={LinkedInSvg}
        iconText="linkedin"
        contact="gabrielfmpires"
      />
    </AboutMeWrapper>
  );
};

export default AboutMe;
