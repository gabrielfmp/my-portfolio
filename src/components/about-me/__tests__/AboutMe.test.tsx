import { render } from '@testing-library/react';
import AboutMe from '../AboutMe';
import React from 'react';

test('snapshot', () => {
  const { asFragment } = render(<AboutMe />);

  expect(asFragment()).toMatchSnapshot();
});
