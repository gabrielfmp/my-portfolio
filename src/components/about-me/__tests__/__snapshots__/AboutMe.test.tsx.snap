// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-g09kig"
  >
    <h2
      class="css-2u4su"
    >
      About me
    </h2>
    <div
      class="css-gqh2np"
    >
      <div
        class="css-47j6j9"
      >
        <img
          alt="email"
          class="css-12k99am"
          src="email.svg"
        />
      </div>
      <div
        class="css-un0t8r"
      >
        gabrielfmpires@gmail.com
      </div>
    </div>
    <div
      class="css-gqh2np"
    >
      <div
        class="css-47j6j9"
      >
        <img
          alt="phone number"
          class="css-12k99am"
          src="phone.svg"
        />
      </div>
      <div
        class="css-un0t8r"
      >
        +351 934429387
      </div>
    </div>
    <div
      class="css-gqh2np"
    >
      <div
        class="css-47j6j9"
      >
        <img
          alt="location"
          class="css-12k99am"
          src="location.svg"
        />
      </div>
      <div
        class="css-un0t8r"
      >
        Lisbon, Portugal
      </div>
    </div>
    <div
      class="css-gqh2np"
    >
      <div
        class="css-47j6j9"
      >
        <img
          alt="birthday"
          class="css-12k99am"
          src="cake.svg"
        />
      </div>
      <div
        class="css-un0t8r"
      >
        17 May 1997
      </div>
    </div>
    <div
      class="css-gqh2np"
    >
      <div
        class="css-47j6j9"
      >
        <img
          alt="github"
          class="css-12k99am"
          src="github.svg"
        />
      </div>
      <div
        class="css-un0t8r"
      >
        gabrielfmp
      </div>
    </div>
    <div
      class="css-gqh2np"
    >
      <div
        class="css-47j6j9"
      >
        <img
          alt="linkedin"
          class="css-12k99am"
          src="linkedin.svg"
        />
      </div>
      <div
        class="css-un0t8r"
      >
        gabrielfmpires
      </div>
    </div>
  </div>
</DocumentFragment>
`;
